src/shared/constants.h

```diff
@@ -108,22 +108,22 @@ const char MULTICAST_CHANNELS[MAX_MULTICAST_CHANNELS][MULTICAST_CHANNEL_LENGTH]
 };
 
 // the port robots are listening to for vision and primitives
-const unsigned VISION_PORT    = 42069;
-const unsigned PRIMITIVE_PORT = 42070;
+const short unsigned int VISION_PORT    = 42069;
+const short unsigned int PRIMITIVE_PORT = 42070;
 
 // the port the AI receives msgs from the robot
-const unsigned ROBOT_STATUS_PORT = 42071;
-const unsigned ROBOT_LOGS_PORT   = 42072;
+const short unsigned int ROBOT_STATUS_PORT = 42071;
+const short unsigned int ROBOT_LOGS_PORT   = 42072;
 
 // the port to listen to for what side of the field to defend
 const unsigned DEFENDING_SIDE_PORT = 42073;
 
 // the timeout to recv a network packet
-const unsigned NETWORK_TIMEOUT_MS = 1000;
+const int NETWORK_TIMEOUT_MS = 1000;
 
 // maximum transfer unit of the network interface
 // this is an int to avoid Wconversion with lwip
-const int MAXIMUM_TRANSFER_UNIT_BYTES = 1500;
+const short unsigned int MAXIMUM_TRANSFER_UNIT_BYTES = 1500;
```

Comment:
Do we have a convention on using signed vs unsigned in the code base?

Reply:
Not at the moment, but we have `-Wconversion` warning set as triggering a compiler error, which forces us to be consistent.